src/Main.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        //In der CODERS.BAY arbeiten wir von 8 bis 16 Uhr. Schreibe ein Programm,
+        // dass eine Zahl von der Konsole einliest und ausgibt ob die Stunde in der Arbeitszeit liegt oder nicht.
+
+        //Bonus: von 12 bis 13 Uhr ist immer Mittagspause, gib also in der Zeit aus, dass Mittag ist.
+            int start = 8;
```

Comment:
wenn du hier variablen anlegst, solltest du diese variablen auch irgendwo verwenden - im besten fall in der if-condition der zeile 20